rest-api/src/main/scala/za/co/absa/enceladus/rest_api/models/rest/Paginated.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.rest_api.models.rest
+
+case class Paginated[T](page: Seq[T], offset: Int, limit: Int, truncated: Boolean) //, message: Option[String] = None)
```

Comment:
Is `truncated` the standard name for the hint there are more data?
Probably just me, but gives me the impression, so of the **returned** data were truncated. 🤷

Reply:
Not sure about _standard_, but this is [inspired by how AWS S3 API signifies that paged listing has more data to be retrieved](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjects.html#AmazonS3-ListObjects-response-IsTruncated).

The impression is correct to a certain degree - you request to obtain items of a collection and either you get all the items (when `collection.itemsCount <= page.size`) or the **returned data does get truncated to fit the page** (`collection.itemsCount > page.size`).

Reply:
OK, if AWS uses this keyword, then it's probably good enough and widely accepted 😉